app.py

```
@@ -0,0 +1,148 @@
+def load_data():
+    try:
+        # Check if file exists
+        if not os.path.exists('books.csv'):
+            st.error("books.csv file not found. Please make sure the file is in the same directory as the application.")
+            return None
+        
+        # First, read the CSV file to get the correct number of columns
+        with open('books.csv', 'r', encoding='latin1') as f:
+            reader = csv.reader(f)
+            header = next(reader)
+            num_columns = len(header)
```

Ruff (0.8.2)

Local variable `num_columns` is assigned to but never used

Remove assignment to unused variable `num_columns`

(F841)